src/main/java/uk/gov/hmcts/cmc/claimstore/services/UserRolesService.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.cmc.claimstore.services;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.idam.models.User;
+import uk.gov.hmcts.cmc.claimstore.repositories.UserRolesRepository;
+import uk.gov.hmcts.cmc.domain.models.UserRole;
+import uk.gov.hmcts.cmc.domain.models.UserRoleRequest;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class UserRolesService {
+    private final UserRolesRepository userRolesRepository;
+    private final UserService userService;
+
+    public UserRolesService(
+        UserRolesRepository userRolesRepository,
+        UserService userService
+    ) {
+        this.userRolesRepository = userRolesRepository;
+        this.userService = userService;
+    }
+
+    public List<String> userRoles(String authorisation) {
+        User user = userService.getUser(authorisation);
+        String userId = user.getUserDetails().getId().toString();
+
+        return userRolesRepository.getByUserId(userId)
+            .stream()
+            .map(UserRole::getRole)
+            .collect(Collectors.toList());
+    }
+
+    public void saveRole(UserRoleRequest userRoleRequest, String authorisation) {
```

Comment:
Service should have no knowledge of request. Pass role name as in string form e.g. 
```
saveRole(String role, String authorisation)
```